js/packages/berty-i18n/locale/af-ZA/messages.json

```diff
@@ -112,10 +112,7 @@
 				"draft": ""
 			}
 		},
-		"suggestion-display-name-initial": "",
-		"scan": {
-			"title": ""
-		}
```

Comment:
You don't need to update every file manually, if you remove `scan` from `en-US/message.json`, crowdin will remove it from all the other files.